DEVELOPERS.md

```diff
@@ -505,7 +505,7 @@ We show cloudifying a host from Hetzner, but the principles should work everywhe
     VM_HOST_ID = ""
     default_boot_images = ["ubuntu-noble", "ubuntu-jammy", "debian-12", "almalinux-9"]
 
-    st = Prog::Vm::HostNexus.assemble(VM_HOST_IP, provider_name: "hetzner", location: "hetzner-fsn1", server_identifier: VM_HOST_ID, default_boot_images: default_boot_images)
+    st = Prog::Vm::HostNexus.assemble(VM_HOST_IP, provider_name: "hetzner", location_id: "caa7a807-36c5-8420-a75c-f906839dad71", server_identifier: VM_HOST_ID, default_boot_images: default_boot_images)
```

Comment:
For the most common locations, I recommend we store the uuids in constants in the Location model, and then reference the constant.

Reply:
I had actually changed it everywhere but this. Since it's an md file, I missed it. Thanks for catching.